src/service/viewer-impl.js

```diff
@@ -221,6 +221,10 @@ export class Viewer {
     this.performanceTracking_ = this.params_['csi'] === '1';
     dev.fine(TAG_, '- performanceTracking:', this.performanceTracking_);
 
+    this.shareTrackingIncomingFragment_ = this.params_['share-tracking'];
```

Comment:
@cramforce From my understanding, when the viewer(from search) generates an amp page with some url, it passes some parameters in the hash. Previously I thought the viewer could set the share-tracking value in the url. But the viewer may never have a hash parameter "share-tracking" because the url is pre-calculated without the information of share-tracking fragment. Shall I assume there's no way the viewer could pass the share-tracking fragment directly? If that's the case, I will not read the parameter at all.